library/acl.inc

```diff
@@ -801,35 +801,23 @@
                           ON group_settings.section_id = acl_sections.section_id
                       WHERE
                         group_settings.group_id IN (".$groupPlacemakers.") AND acl_sections.`section_identifier` = ? ";
-    $sql_group_acl_denied  = $sql_group_acl_base . " AND group_settings.allowed = '0'";
+
     $sql_group_acl_allowed = $sql_group_acl_base . " AND group_settings.allowed = '1'";
 
     // Complete the group queries sql binding array
     array_push($groups_sql_param, $section_identifier);
 
-        $count_user_denied      = 0;
-        $count_user_allowed     = 0;
```

Comment:
There is no need for $sql_group_acl_denied
and I changed it to 
    $count_group_allowed    = 0;
    $count_user_allowed     = 0;